src/components/courses/components/CourseInfo/CourseInfo.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect, useState } from 'react';
+import './CourseInfo.css';
+import { CourseInfoProps } from './CourseInfo.types';
+import { Link, useParams } from 'react-router-dom';
+
+const CourseInfo: React.FC<CourseInfoProps> = () => {
+  const { id } = useParams();
+  const [course, setCourse] = useState({});
+
+  useEffect(() => {
+    const getCourse = fetch('http://localhost:4000/courses/' + id, {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    });
+
+    getCourse
+      .then((response) => response.json())
+      .then((data) => setCourse(data.result));
```

Comment:
Let's stick to the `await/async` syntax for the sake of consistency. I don't see an error handling as well.